packages/components/src/VirtualizedList/CellIconText/CellIconText.component.js

```diff
@@ -0,0 +1,42 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import Icon from '../../Icon';
+import { getTheme } from '../../theme';
+import theme from './CellIconText.scss';
+
+const css = getTheme(theme);
+
+/**
+ * Cell renderer that displays a boolean
+ */
+class CellIconText extends React.Component {
+	shouldComponentUpdate(nextProps) {
+		return this.props.cellData !== nextProps.cellData;
+	}
+
+	render() {
+		const { cellData } = this.props;
+
+		return (
+			<div className={css('tc-virtualizedlist-cell-icon-text')}>
+				<Icon name={cellData.icon} />
+				<span className={css('tc-virtualizedlist-cell-icon-text-label')}>{cellData.label}</span>
```

Comment:
with css modules, this classname will be something like `tc-virtualizedlist-cell-icon-text__tc-virtualizedlist-cell-icon-text-label__hash`

I think you should have a sass like this
```
.tc-virtualizedlist-cell-icon-text {
    .label {}
}
```
And in the js code 
```
<span className={css.label}>
```